src/Raven.Studio/typescript/common/queryCompleter.ts

```diff
@@ -453,7 +453,10 @@ class queryCompleter {
         callback(null,  keywords.map(keyword  => {
             const word = <autoCompleteWordList>keyword;
             word.caption = _.trim(keyword.value, "'");
-            keyword.value += " ";
+            if (keyword.value.includes(" ")){
```

Comment:
Reverse the logic, if the collection name isn't just A-Z, 0-9 _, then we need qoutes.

Reply:
What other values should be escaped? (other than eascape characters, which contains `\`)
Collection names in other languages are just working.

Reply:
IsLetterOrDigit ?

Reply:
Added to the todo list.

Reply:
Done, currently only for English words but we might add more to the regex:
https://github.com/fitzchak/ravendb/blob/v4.0/src/Raven.Studio/typescript/common/queryCompleter.ts#L470